applications/crossbar/src/modules/cb_port_requests.erl

```diff
@@ -427,7 +427,19 @@ post(Context, Id, ?PORT_SUBMITTED) ->
 post(Context, Id, ?PORT_SCHEDULED) ->
     do_post(Context, Id);
 post(Context, Id, ?PORT_COMPLETE) ->
-    do_post(Context, Id);
+    try send_ported_notification(Context, Id) of
+        _ ->
+            lager:debug("ported notification sent"),
+            do_post(Context, Id)
+    catch
+        _E:_R ->
+            lager:debug("failed to send the ported notification: ~s:~p", [_E, _R]),
+            cb_context:add_system_error(
+                'bad_gateway'
+                ,<<"failed to send ported email to system admins">>
```

Comment:
Do we want to fail the request if the email fails to send? I don't know at the moment. What's your thought process here?